extensions/amp-font/0.1/fontloader.js

```diff
@@ -160,13 +160,11 @@ export class FontLoader {
           } else if (this.compareMeasurements_()) {
             resolve();
           } else {
-            vsync.run(vsyncTask);
```

Comment:
For now, yes. `run` - one time call. `createTask` - repeated calls. This way they are easy to tell apart.